README.md

```diff
@@ -39,41 +41,67 @@ we would like things to flow.
 
-`docker build -t nexus-repository-conan:0.0.6 .`
+[We have detailed instructions on how to get started here!](https://help.sonatype.com/repomanager3/formats/conan-repositories)
 
-#### Run as a Docker container
+## Compatibility with Nexus Repository Manager 3 Versions
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
+The table below outlines what version of Nexus Repository Manager the plugin was built against:
 
-For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+| Plugin Version    | Nexus Repository Manager Version |
+|-------------------|----------------------------------|
+| v0.0.1            | <3.11.0                          |
+| v0.0.2 - v0.0.6   | >=3.11.0                         |
+| v1.0.0 In product | >=3.22.0                         |
 
-The application will now be available from your browser at http://localhost:8081
 
-## Using Conan With Nexus Repository Manager 3
+## Installing the plugin
 
-[We have detailed instructions on how to get started here!](docs/CONAN_USER_DOCUMENTATION.md)
+In Nexus Repository Manager 3.22+ Conan proxy is already included, so there is no need to install it.
+But if you want to reinstall the plugin with your improvements then the following instructions will be useful.
+Note: Using an unofficial version of the plugin is not supported by the Sonatype Support team.
 
-## Installing the plugin
+### Permanent Reinstall
+
+* Copy the bundle into: `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-conan/1.0.0/nexus-repository-conan-1.0.0.jar`
+* Modify xml to introduce the plugin:
+  * OSS edition: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
+
+  * PRO edition: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
+
+   ```
+         <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
+         <feature version="1.0.0" prerequisite="false" dependency="false">nexus-repository-conan</feature>
+         <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
+     </features>
+   ```
+
+   And
+   ```
+       <feature name="nexus-repository-conan" description="org.sonatype.nexus.plugins:nexus-repository-conan" version="1.0.0">
+        <details>org.sonatype.nexus.plugins:nexus-repository-conan</details>
+        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-conan/1.1.0</bundle>
+       </feature>
+    </features>
+   ```
 
-There are a range of options for installing the Conan plugin. You'll need to build it first, and
+### Installing the plugin in old versions
+
+For older versions there are a range of options for installing the Conan plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
```

Comment:
Trivial suggestion: This hasn't changed, but suggesting we change this wording from "the options shown below" to either "one of the options shown below" or "an option shown below" since it's just one.  As written it sounds like you use them all =)

Reply:
fixed 9256cd94a7e43a6f43ad6f942b7665f0468ccb7d